src/applications/claims-status/actions/index.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import * as Sentry from '@sentry/browser';
 
-import recordEvent from 'platform/monitoring/record-event';
-import { apiRequest } from 'platform/utilities/api';
-import get from 'platform/utilities/data/get';
-import environment from 'platform/utilities/environment';
+import { recordEvent } from '@department-of-veterans-affairs/platform-monitoring/exports';
+import { apiRequest } from '@department-of-veterans-affairs/platform-utilities/exports';
+import get from '@department-of-veterans-affairs/platform-forms-system/get';
+import environment from '@department-of-veterans-affairs/platform-utilities/environment';
 import localStorage from 'platform/utilities/storage/localStorage';
```

Comment:
I skipped this one because I couldn't figure out what the import was supposed to be